src/shogun/machine/Machine.h

```diff
@@ -333,6 +333,18 @@ class CMachine : public CStoppableSGObject
 			return false;
 		}
 
+		/** Continue Training
```

Comment:
This need proper documentation pls here as well, this is user facing.

Reply:
I.e. explain what the method does (from a user perspective)

Something in the lines of:
"Continue training.

This method can be used to continue a previously prematurely stopped training call.
It throws an error if the machine does not support this feature.
"

No talking about overloading (that would be developer facing)